raiden_contracts/deploy/etherscan_verify.py

```diff
@@ -0,0 +1,191 @@
+import json
+import click
+import requests
+from time import sleep
+from pathlib import Path
+
+from eth_abi import encode_abi
+
+from raiden_contracts.constants import (
+    CONTRACT_ENDPOINT_REGISTRY,
+    CONTRACT_SECRET_REGISTRY,
+    CONTRACT_TOKEN_NETWORK_REGISTRY,
+)
+from raiden_contracts.contract_manager import (
+    ContractManager,
+    contracts_precompiled_path,
+    contracts_source_path,
+    get_contracts_deployed,
+)
+
+
+@click.command()
+@click.option(
+    '--chain-id',
+    default=3,
+    help='Chain id. E.g. --chain-id 3',
+)
+@click.option(
+    '--apikey',
+    required=True,
+    help='A valid Etherscan APIKEY is required.',
+)
+@click.option(
+    '--guid',
+    help='GUID from a previous verification attempt. Tries to get the submission status.',
+)
+@click.option(
+    '--contract-name',
+    default=None,
+    help='Contract name. Options: EndpointRegistry | SecretRegistry | TokenNetworkRegistry',
+)
+def etherscan_verify(chain_id, apikey, guid, contract_name):
+    contract_manager = ContractManager(contracts_precompiled_path())
+    source_path = contracts_source_path()
+
+    deployment_info = get_contracts_deployed(chain_id)
+
+    if chain_id == 3:
+        etherscan_api = 'https://api-ropsten.etherscan.io/api'
+    elif chain_id == 4:
+        etherscan_api = 'https://api-rinkeby.etherscan.io/api'
+    elif chain_id == 42:
+        etherscan_api = 'https://api-kovan.etherscan.io/api'
+    elif chain_id == 1:
+        etherscan_api = 'https://api.etherscan.io/api'
```

Comment:
Style nitpick:
I think for this we can use a dictionary where:
```
{
3: "'https://api-ropsten.etherscan.io/api'",
4: "https://api-rinkeby.etherscan.io/api",
... etc
}
```
And use `.get(chain_id)` to fetch the URL that needs to be called.